src/Form/ChoiceList/ModelChoiceLoader.php

```diff
@@ -17,89 +17,89 @@
 use Sonata\AdminBundle\Model\ModelManagerInterface;
 use Sonata\Doctrine\Adapter\AdapterInterface;
 use Symfony\Component\Form\ChoiceList\ArrayChoiceList;
+use Symfony\Component\Form\ChoiceList\ChoiceListInterface;
 use Symfony\Component\Form\ChoiceList\Loader\ChoiceLoaderInterface;
 use Symfony\Component\Form\Exception\RuntimeException;
-use Symfony\Component\PropertyAccess\PropertyAccess;
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
-use Symfony\Component\PropertyAccess\PropertyPath;
 
 /**
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
 final class ModelChoiceLoader implements ChoiceLoaderInterface
 {
     /**
-     * @var \Sonata\AdminBundle\Model\ModelManagerInterface
+     * @var ModelManagerInterface
      */
     private $modelManager;
 
     /**
-     * @var string
+     * @var string|null
+     * @phpstan-var class-string|null
      */
     private $class;
 
+    /**
+     * @var string|null
+     */
     private $property;
```

Comment:
This property has no (internal) usage and could be removed

Reply:
Fixed